collect_sources.sh

```diff
@@ -63,12 +63,11 @@ fetch_deb_src() {
 
 # exclude binary-only Raspbian packages
 EXCLUDE='realvnc-vnc'
+EXCLUDE+='|widevine'
```

Comment:
A bit more information in the commit would be nice.
https://github.com/RPi-Distro/pi-gen/blob/01d24ef22778337ed04cf9d6444b1be57b6a1e1a/stage3/00-install-packages/00-packages#L8